FirebaseRemoteConfig/Tests/Unit/RCNTestUtilities.m

```diff
@@ -39,11 +39,15 @@ + (NSString *)generatedTestAppNameForTest:(NSString *)testName {
 
 /// Remote Config database path for test version
 + (NSString *)remoteConfigPathForTestDatabase {
-  NSArray<NSString *> *dirPaths =
+#if TARGET_OS_TV
+  NSArray *dirPaths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
+#else
+  NSArray *dirPaths =
```

Comment:
nit: It seems like this condition appears in the code enough times to justify creating a method in core or google utilities to encapsulate it. Not necessary should be a part of this PR.

Reply:
I think it's a reasonable idea, I do hesitate because I also want to make sure people are making intentional decisions about where they store things and recognize that this could be writing to a cache. Ideally I'd like to see this pop up in danger or something - if you're running tvOS tests and the active target has the `NSApplicationSupportDirectory` symbol, throw a warning/error.